applications/crossbar/doc/ref/braintree.md

```diff
@@ -8,6 +8,16 @@
 
 #### Fetch
 
+> GET /v2/accounts/{ACCOUNT_ID}/braintree/client_token
```

Comment:
This should be added to doc/braintree.md (or create that doc if missing) and fleshed out with example queries and responses.